control.log

```diff
@@ -0,0 +1 @@
+[08-04|12:36:23.201] INFO control/control.go:221 [0;32mstart response: cluster_info:{pid:380100  root_data_dir:"/tmp/network-runner-root-data_20220804_123623"}[0;0m
```

Comment:
this should be removed

Reply:
is this new stuff generating log files? for that case, we need to control the output paths

Reply:
Yeah so apparently instantiating a logger will produce a log file. If no log dir has been set via `config.LogDirectory`, then the current directory appears to be used.

The way to set no logging is to set the `LogLevel = logging.Off`. Then the `DisplayLevel` will print to screen. 

I have now used this approach where I believed no log file is necessary (client, control) - also to avoid a multitude of spread out log files. Whereas the server part has configurable log dir with its flag.